scripts/filter_project.js

```diff
@@ -1,25 +1,29 @@
-/*global $*/
-$(function () {
+/* global */
+window.addEventListener('DOMContentLoaded', function() {
     "use strict";
-    var langSelector = $("#language");
-    var langMatch = $("#lang-match");
-    var diffAll = $("#diff-all");
-    var diffOpt = $(".diff-opt");
+    var langSelector = document.getElementById("language");
+    var langMatch = document.getElementById("lang-match");
+    var diffAll = document.getElementById("diff-all");
+    var diffOpt = document.querySelectorAll(".diff-opt");
 
     function showMatcher() {
-        langMatch.css("display", (0 === langSelector.prop("selectedIndex") ? "none" : "block"));
+        langMatch.style.display = 0 === langSelector.selectedIndex ? "none" : "block";
     }
 
-    langSelector.change(function () {
+    langSelector.addEventListener("change", function () {
         showMatcher();
     });
 
-    diffAll.change(function () {
-        diffOpt.prop("checked", false);
+    diffAll.addEventListener("change", function () {
+        diffOpt.forEach(diffOption => {
+            diffOption.checked = false;
+        });
     });
 
```

Comment:
The blocks below and above were ones where I (as a js/jQuery noob) couldn't see the reason for the differences. One has foreach inside the function, and one outside, and the original didn't have foreach at all. No need for an explanation - just checking it is what you want.

Reply:
I'd like someone to check me on this. I don't know whether it's in `tools/project_manager/show_word_context.js` or `tools/project_manager/show_word_context.php`, but I'm seeing a problem with the image display. If I click on the first image listed in the context list, it displays fine. But all subsequent image links do not work. The test project I'm using is `projectID45e0c521f24cd`.

Reply:
> I'd like someone to check me on this. ...

@chrismiceli and I looked at this via Slack -- he could not reproduce the problem, and upon re-testing, neither could I. I've no idea what was going on unless it was that I was also testing on TEST.

Reply:
> Apologies for replying to an unrelated comment above when I should have started a new comment above.
> 
> > > I'd like someone to check me on this. ...
> > 
> > 
> > @chrismiceli and I looked at this via Slack -- he could not reproduce the problem, and upon re-testing, neither could I. I've no idea what was going on unless it was that I was also testing on TEST.
> 
> I went back and tested again, and realized that when Chris and I were not seeing the problem, that was on TEST, not in the branch sandbox. How I got there:
> 
>     * Edit project word lists ->
> 
>     * Suggestions from proofreaders: Display ->
> 
>     * Context
> 
> 
> I've duplicated the same result as before on two projects. That is, the first context link works. Any link after that doesn't.



> Apologies for replying to an unrelated comment above when I should have started a new comment above.
> 
> > > I'd like someone to check me on this. ...
> > 
> > 
> > @chrismiceli and I looked at this via Slack -- he could not reproduce the problem, and upon re-testing, neither could I. I've no idea what was going on unless it was that I was also testing on TEST.
> 
> I went back and tested again, and realized that when Chris and I were not seeing the problem, that was on TEST, not in the branch sandbox. How I got there:
> 
>     * Edit project word lists ->
> 
>     * Suggestions from proofreaders: Display ->
> 
>     * Context
> 
> 
> I've duplicated the same result as before on two projects. That is, the first context link works. Any link after that doesn't.

I can reproduce it and know what is happening. I'll send a commit tonight to resolve this. Essentially I attached a click handler to the first image, but not the others. This was because the selector selected multiple items and I acted like it only selected one. I'll switch to a querySelectorAll and attach an event listener to each image link.